### PR TITLE
Added an initial setup instructions section

### PR DESCRIPTION
**Pull request by utkut**
I have added a simple instructions section based on the information from the hackaday project page. There are some of the install details missing, but this might be a good start for other users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,51 @@
...
+
+Installation for python3-pip, raspotify, python3-tk, 
+```
+sudo apt install python3-pip
+
+sudo curl -sL https://dtcooper.github.io/raspotify/install.sh | sh
+
+sudo apt-get install python3-tk 
+
+sudo apt-get install python3-pigpio
```
Comment (dupontgu) [inline README.md:23]: Don't actually use pigpio from python, you can delete this line. Good thought though.

File: README.md [hunk 2]
```diff
@@ -2,4 +2,51 @@
...
+```
+git clone https://github.com/bablokb/pi-btaudio.git
+cd pi-btaudio
+sudo tools/install
+```
+
+5. Install Openbox
+```
+git clone git://git.openbox.org/mikachu/openbox openbox
+cd openbox
```
Comment (dupontgu) [inline README.md:44]: Looks like there's a step missing? Also, I believe I simply used apt to install openbox. Totally cool to link to the source though

**Review by dupontgu — CHANGES_REQUESTED** [review]
Some tiny things! Thanks so much for doing this!

**Review by utkut — COMMENTED** [review]
Thanks for adding the missing steps.

**Review by dupontgu — APPROVED** [review]
This is awesome, thanks!

**Comment by rsappia** [comment]
silly point, but missing :

`sudo apt-get install git`

**Pull request merged**